quantum/quantum.c

```diff
@@ -103,6 +103,56 @@ bool process_record_quantum(keyrecord_t *record) {
           return false;
       }
       break;
+	#ifdef RGBLIGHT_ENABLE
+	case RGB_TOG:
+		if (record->event.pressed) {
+			rgblight_toggle();
+			return false;
```

Comment:
So then should it look like:

`}`
`return false;`
`break;`
?

And if so, do you want me to fix RESET and DEBUG? They have the same mistake - they were what I used as examples for my code to look like.